constants_manager/constants_manager.py

```diff
@@ -7,19 +7,15 @@
 
 class ConstantsManager():
     def __init__(self, config_file_name='constants.ini', constants_name='ENV'):
-        self.config_file_name = config_file_name
-        self.constants_name = constants_name
+        self.config = configparser.RawConfigParser()
+        self.config.optionxform = str
+        self.config.read(config_file_name)
+        self.environment = 'DEFAULT'
+        if constants_name in os.environ:
+            self.environment = os.environ[constants_name]
```

Comment:
Why it is treated in constructor? I don't like the process is existing in constructor.
I think it would be better that we prepared it as a method. What would you think about it?